airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/source/relationaldb/RelationalDbQueryUtils.java

```diff
@@ -23,6 +31,21 @@ public class RelationalDbQueryUtils {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(RelationalDbQueryUtils.class);
 
+  public record TableSizeInfo(Long tableSize, Long avgRowLength) {}
+
+  public static final String TABLE_SIZE_BYTES_COL = "TotalSizeBytes";
+  public static final String AVG_ROW_LENGTH = "AVG_ROW_LENGTH";
+
+  public static final String TABLE_ESTIMATE_QUERY = """
+                                                     SELECT
+                                                       (data_length + index_length) as %s,
+                                                       AVG_ROW_LENGTH as %s
+                                                    FROM
+                                                       information_schema.tables
+                                                    WHERE
+                                                       table_schema = '%s' AND table_name = '%s';
+                                                    """;
+
```

Comment:
Wasn't all of this defined in the previous PR in the stack?

Reply:
I removed this extraneous code as we already have it in the mssql specific query utils